src/steamship/utils/kv_store.py

```diff
@@ -1,9 +1,11 @@
 """A simple key-value store implemented atop Files and Tags."""
 
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional, Tuple
 
 from steamship import Block, File, Steamship, Tag
 
+KV_STORE_MARKER = "__init__"
+
 
 class KeyValueStore:
```

Comment:
Just sharing: 

In case you want to support accessing the kv store like a dict (`my_kv["key"] = my_value` and `my_kv["key"]`) we only need to implement two functions: 

```
    def __getitem__(self, key):
    def __setitem__(self, key, value):
```

Reply:
Bonus for supporting `del my_kv["key"]`: 
```
    def __delitem__(self, key):
```

Reply:
Oh that's nice. I'm a *bit* hesitant to do it here since the get/set/delete involves HTTP calls over the network and for some reason having builtin `[]` operators backed by an HTTP call feels funny -- but appreciate the tip.